expression/builtin_json.go

```diff
@@ -50,27 +50,16 @@ var (
 	_ functionClass = &jsonKeysFunctionClass{}
 	_ functionClass = &jsonLengthFunctionClass{}
 
-	// Type of JSON value.
```

Comment:
Any reason to delete the comments?

Reply:
reasons as following:

 - they are useless,  see [here](https://github.com/pingcap/tidb/blob/master/expression/builtin_math.go#L67)
- they cause gowrod error
- i don't know how to avoid gowrod error keeping this commens
- @shenli and i had discussed [here](https://github.com/pingcap/tidb/pull/7713#discussion_r217902463)